pkg/operator/status.go

```diff
@@ -71,8 +71,6 @@ func (optr *Operator) statusProgressing() error {
 
 	conds := []osconfigv1.ClusterOperatorStatusCondition{
 		newClusterOperatorStatusCondition(osconfigv1.OperatorProgressing, isProgressing, string(ReasonSyncing), message),
-		newClusterOperatorStatusCondition(osconfigv1.OperatorAvailable, osconfigv1.ConditionTrue, "", ""),
-		newClusterOperatorStatusCondition(osconfigv1.OperatorDegraded, osconfigv1.ConditionFalse, "", ""),
 		operatorUpgradeable,
```

Comment:
don't we need to do the same for `statusAvailable`?

Reply:
We chatted about this offline, but following up here: The API is a little weird, but the way we use this means that `statusProgressing` and `statusDegraded` are a little different than `statusAvailable`. The first two should only modify their condition, while `statusAvailable` specifically means that we are not degraded and not progressing and we *are* available. This is noted (briefly) in the godoc comments for the methods.